src/components/reference/_triage-states.mdx

```diff
@@ -0,0 +1,11 @@
+| Status | Description |
+| -----------  | ------------ |
+| **Open** | Findings are open by default. A finding is open if it was present last time when Semgrep scanned the code and finding's triage state was not marked as ignored. An open finding represents a match between the code and a rule that is enabled in the repository. Open findings require action, such as rewriting the code to eliminate the detected vulnerability. |
+| **Ignored** | Findings that are ignored are present in the code, but have been labeled as unimportant. Ignored findings can be labeled that way because they are false positives or because they have been deprioritized. You can mark findings as ignored through Semgrep App (see [Managing finding status](/semgrep-app/findings/#managing-finding-status-bulk-triage)) or by adding a code comment (see [nosemgrep comments](/ignoring-files-folders-code/#reference-summary)). |
+| **Fixed** | <p>Fixed findings were detected in a previous scan, but are no longer detected in the most recent scan of that same branch. The Semgrep rule that matched the finding and the filepath where the finding was located must both still be present for the finding to be marked as fixed. </p> <p> Change the state of a finding to **fixed** by: <ol type="a"><li>Fixing the code so the rule does not match it.</li><li>Editing the Semgrep rule so it no longer matches the code.</li></ol></p> |
+
+Findings can also be **removed**. A removed finding does not count towards a fix rate or the total number of findings, and does not appear in the Semgrep App. A finding is removed if it is not found in the most recent scan of the branch where it was detected due to any of the following conditions:
```

Comment:
```suggestion
Findings can also be **removed**. A removed finding does not count towards fix rates or finding counts, and does not appear in the Semgrep App. A finding is removed if it is not found in the most recent scan of the branch where it was detected due to any of the following conditions:
```

Reply:
I think the total number of findings was less ambiguous. When I read finding counts, I stopped and I was thinking for a while what are we talking about? @chmccreery